src/khoj/processor/conversation/openai/gpt.py

```diff
@@ -125,6 +127,13 @@ def converse(
 
     conversation_primer = prompts.query_prompt.format(query=user_query)
 
+    system_prompt = ""
```

Comment:
nit: This initialization isn't required as system_prompt is always set in the if/else conditional below